### PR TITLE
Project 27 - Update MARS documentation

### PR DESCRIPTION
**Pull request by M-casado**
- [x] Reviewed documentation
- [x] Added functions and responsibilities
- [x] Added Archives as a role

### CODE REVIEW

File: 27/mars_documentation.md [hunk 1]
```diff
@@ -10,19 +10,41 @@ MARS is comprised of distinct components, each representing key roles in the dat
...
-* Function: Prepare research data in ISA-JSON format, generate and export ISA-JSON files.
+This role includes any source of ISA-JSON files containing the metadata of multi-omics research. For example, research infrastructure or service facilities that operate platforms such as ARC, FAIRDOM-SEEK, ISA Creators. 
+
+* Main function: Prepare research metadata in ISA-JSON format (i.e. generate and export ISA-JSON files)
 * Responsibilities:
-  * Ensure compatibility of the exported ISA-JSON files with the format expected by MARS.
-  * Support and validate for the use of repositories’ checklists, controlled vocabulary, ontology terms, and other required metadata.
+  * Avoid data duplication. For example, not re-submitting already existing data in the target repositories through MARS.
+  * Ensure compliance of the submitted ISA-JSON with the ISA model expected by MARS.
+  * Conform submitted ISA-JSON to target repositories' additional validation rules validated by MARS. This includes repositories’ checklists, controlled vocabulary lists, ontology terms, etcetera.
```
Comment (floradanna) [inline 27/mars_documentation.md:19]: Is MARS going to do the validation of checklists etc? I thought that would be done by target archives during submission?

**Comment by floradanna** [comment]
I would like to add the role of "end user": the one who input the data in platform that produces ISA-JSON. OK for you?

File: 27/mars_documentation.md [hunk 2]
```diff
@@ -10,19 +10,41 @@ MARS is comprised of distinct components, each representing key roles in the dat
...
+  * Assert that files referenced within the ISA-JSON metadata are uploaded to the target repositories. For example, if a FASTQ file is referenced in an assay that would be submitted to ENA, this file should exist within the storage of ENA.
```
Comment (proccaserra) [inline 27/mars_documentation.md:20]:
```suggestion
  * Assert that files referenced within the ISA-JSON metadata are uploaded to the target repositories. For example, if a FASTQ file is referenced in an assay that would be submitted to ENA, this file should exist within the storage of ENA. The ISA-JSON metadata payload should contain the checksums associated to the data files (this is requirement for SRA based repositories)
```

File: 27/mars_documentation.md [hunk 3]
```diff
@@ -10,19 +10,41 @@ MARS is comprised of distinct components, each representing key roles in the dat
 Each role performs specific functions and carries distinct responsibilities essential for the smooth operation of the system.
```
Comment (floradanna) [inline 27/mars_documentation.md:11]:
```suggestion
### End-User
This individual is the user who inputs experimental metadata and location of related data files into ISA-JSON Producer platforms. End-User provides essential experimental details that are encoded into ISA-JSON format.

End-User could be also direct user of the Data Broker platform, particularly in scenarios where the broker platform is only accessible as a web service within the institute's intranet.

Main function: 
* Input experimental metadata into ISA-JSON Producer platforms following the specific rules, structures, and requirements of the respective platform. 
* Use the ISA-JSON producer platfom to expor experimental metadata in ISA-JSON files and uploading them to the Data Broker platform.
Responsibilities:
* End-User ensures the accuracy and completeness of metadata.
* End-User must refrain from uploading identical ISA-JSON files to the Data Broker platform, unless it is to initiate a distinct action such as update or release.
```

File: 27/mars_documentation.md [hunk 4]
```diff
@@ -10,19 +10,41 @@ MARS is comprised of distinct components, each representing key roles in the dat
...
 ### ISA-JSON producers
-This role includes research infrastructure or service facilities that operate platforms such as ARC, FAIRDOM-SEEK, ISA Creators or any other tool that generates ISA-JSON files.
-* Function: Prepare research data in ISA-JSON format, generate and export ISA-JSON files.
+This role includes any source of ISA-JSON files containing the metadata of multi-omics research. For example, research infrastructure or service facilities that operate platforms such as ARC, FAIRDOM-SEEK, ISA Creators. 
+
+* Main function: Prepare research metadata in ISA-JSON format (i.e. generate and export ISA-JSON files)
 * Responsibilities:
-  * Ensure compatibility of the exported ISA-JSON files with the format expected by MARS.
-  * Support and validate for the use of repositories’ checklists, controlled vocabulary, ontology terms, and other required metadata.
+  * Avoid data duplication. For example, not re-submitting already existing data in the target repositories through MARS.
```
Comment (floradanna) [inline 27/mars_documentation.md:17]:
```suggestion
```

File: 27/mars_documentation.md (hunk 1 above)
Comment (floradanna) [inline 27/mars_documentation.md:19]:
```suggestion
  * Validate ISA-JSON structure and fields in ISA-JSON as expected by MARS (e.g. “Comment[Target repository]” at Assay level). This includes verification of the checksums associated to the data files. This step might also include validation of repositories’ checklists, controlled vocabulary lists, ontology terms etc, if validation rules are provided by each repository for MARS.
```

**Review by floradanna — CHANGES_REQUESTED** [review]

File: 27/mars_documentation.md [hunk 5]
```diff
@@ -8,21 +8,52 @@ MARS serves as a common ground where data producers and diverse data repositorie
...
 Each role performs specific functions and carries distinct responsibilities essential for the smooth operation of the system.
+### End-User
+This individual is the user who inputs experimental metadata and location of related data files into ISA-JSON Producer platforms. End-User provides essential experimental details that are encoded into ISA-JSON format.
 
+End-User could be also direct user of the Data Broker platform, particularly in scenarios where the broker platform is only accessible as a web service within the institute's intranet.
+
+Main function: 
+* Input experimental metadata into ISA-JSON Producer platforms following the specific rules, structures, and requirements of the respective platform. 
+* Use the ISA-JSON producer platfom to expor experimental metadata in ISA-JSON files and uploading them to the Data Broker platform.
+Responsibilities:
```
Comment (floradanna) [inline 27/mars_documentation.md:19]:
```suggestion

Responsibilities:
```

**Review by floradanna — APPROVED** [review]

**Pull request merged**